src/libraries/System.Net.Requests/src/System/Net/ServicePoint/BindIPEndPoint.cs

```diff
@@ -3,5 +3,6 @@
 
 namespace System.Net
 {
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System.Net.ServicePoint, Version=8.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51")]
```

Comment:
This shouldn't be necessary. TypeForwardedFrom attributes are only necessary for binary serialization which these types don't support.